src/java/org/apache/cassandra/db/lifecycle/Tracker.java

```diff
@@ -604,6 +604,14 @@ private void notify(INotification notification)
             subscriber.handleNotification(notification, this);
     }
 
+    public void publishMetrics(MetricsNotification metricsNotification)
+    {
+        for (INotificationConsumer subscriber : subscribers)
```

Comment:
Nit: why not just `notify(metricsNotification)`?

Note: one-liner methods are usually immediately inlined, so there isn't even a performance penalty for doing that.